test/e2e/framework/util.go

```diff
@@ -1311,7 +1311,7 @@ func DumpNodeDebugInfo(c clientset.Interface, nodeNames []string, logFunc func(f
 
 // getKubeletPods retrieves the list of pods on the kubelet.
 func getKubeletPods(c clientset.Interface, node string) (*v1.PodList, error) {
-	var result *v1.PodList
```

Comment:
I suppose the fix is okay (don't know the code).

While you are at it, can you move this line down closer to where `result` is actually used?

https://github.com/kubernetes/kubernetes/blob/47f450a960e89876fd481f103f403f2fcd7e9838/test/e2e/framework/util.go#L1330

Reply:
Moved :-)